### PR TITLE
Feature : About page

### PR DESCRIPTION
**Pull request by justinehell**
Add About page and footer

### CODE REVIEW

File: src/i18n/en-US/index.js [hunk 1]
```diff
@@ -313,6 +313,29 @@ export default {
         filterParsableFiles: 'Show parsable files',
       },
     },
+    about: {
+      title: 'About Leto Modelizer',
+      letoModelizer: {
+        title: 'What is Leto Modelizer ?',
+        description: 'It is a low code modelizer that works with external plugins for technology integration (terraform, jenkins, kubernetes...). The goal of Leto Modelizer is to become the standard tool for all developers to easily reuse and exploit the skills of architecture and/or DevOps experts.',
```
Comment (k-y0u) [inline src/i18n/en-US/index.js:320]: `technology integration` -> `integration technology` ?

File: src/pages/AboutPage.vue [hunk 2]
```diff
@@ -0,0 +1,236 @@
+<template>
+  <q-layout
+    class=""
```
Comment (k-y0u) [inline src/pages/AboutPage.vue:3]: no class in `class`

File: src/layouts/MainLayout.vue [hunk 3]
```diff
@@ -1,15 +1,41 @@
 <template>
-  <q-layout view="lHh Lpr lFf">
+  <q-layout view="lHh Lpr lff">
```
Comment (k-y0u) [inline src/layouts/MainLayout.vue:2]: ATTENTION ! This could change the structure of other pages

File: src/layouts/MainLayout.vue [hunk 4]
```diff
@@ -1,15 +1,41 @@
...
     <q-page-container>
       <router-view />
     </q-page-container>
+    <q-footer
```
Comment (k-y0u) [inline src/layouts/MainLayout.vue:6]: not agree (let's talk about it)

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/i18n/en-US/index.js (hunk 1 above)
Comment (justinehell) [reply src/i18n/en-US/index.js:320]: done

File: src/pages/AboutPage.vue (hunk 2 above)
Comment (justinehell) [reply src/pages/AboutPage.vue:3]: oops done

File: changelog.md [hunk 5]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+* Create About page and footer
```
Comment (k-y0u) [inline changelog.md:12]: missing dot

**Review by k-y0u — APPROVED** [review]

**Review by k-y0u — DISMISSED** [review]

**Review by k-y0u — APPROVED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Pull request merged**